l10n_tools/extract.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 cd ..
-pybabel extract -F babel.cfg -o locale/templates/LC_MESSAGES/messages.pot .
-cd website 
-cd ../l10n_tools
+pybabel extract -F babel.cfg -o libs/locale/templates/LC_MESSAGES/messages.pot .
+cd sites/website
+cd ../../l10n_tools
```

Comment:
I assume I can remove the first cd, but I'd rather not accidentally break something that was working before.